pkgs/tools/misc/vmtouch/default.nix

```diff
@@ -2,14 +2,14 @@
 
 stdenv.mkDerivation rec {
   pname = "vmtouch";
-  version = "git-20150310";
+  version = "1.0.2";
```

Comment:
I realized this too when I did the upgrade.  It seems that it is the convention to put `-git` as part of the name for packages that come from git (there are numerous examples of this, and is why I did it originally).  Given the circumstances, is there a better solution than what I've done with this upgrade?